validator_client/src/notifier.rs

```diff
@@ -48,6 +50,8 @@ async fn notify<T: SlotClock + 'static, E: EthSpec>(
             "available" => num_available,
             "synced" => num_synced,
         )
+    } else if num_synced > 1 {
```

Comment:
I guess this isn't strictly true since we might have a connected fallback but a disconnected primary.

Reply:
Fixed this here d55da59